extensions/binding/org.eclipse.smarthome.binding.bluetooth/src/main/java/org/eclipse/smarthome/binding/bluetooth/ble/GattService.java

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (c) 1997, 2015 by Huawei Technologies Co., Ltd. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.bluetooth.ble;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.eclipse.smarthome.binding.bluetooth.BluetoothBindingConstants;
+
+public enum GattService {
```

Comment:
in io.transport.bluetooth we already have BluetoothGattService and BluetoothGattCharacteristic - this feels a bit duplicated here (although the content is different). Would it make sense to move this to io.transport as well, especially as it is the only class (enum) in this package?